src/app/modules/account/pages/personal-info/personal-details/personal-details.component.html

```diff
@@ -19,7 +19,14 @@ <h4 [innerHTML]="mspAccountApp.accountChangeOptions.personInfoUpdate ? lang('./e
         </div>
     </div>
 
+<<<<<<< HEAD:src/app/components/msp/account/personal-info/personal-details/personal-details.component.html
```

Comment:
Looks like there's a merge conflict here.  You should resolve any merge conflicts before pushing them up.  In general, you should try and never commit a merge conflict.  This one may be commented out, but point stands and it should be removed.